eng/pipelines/docs.yml

```diff
@@ -2,85 +2,103 @@ trigger:
   - master
 jobs:
   - job: CreateDocs
-    variables:
-      skipComponentGovernanceDetection: true
     pool:
       vmImage: vs2017-win2016
     steps:
+      # First set of steps download and install tools or create directories to be used in doc generation
+      - powershell: |
+          Invoke-WebRequest -Uri "https://github.com/mono/api-doc-tools/releases/download/mdoc-5.7.4.9/mdoc-5.7.4.9.zip" `
+          -OutFile "mdoc.zip" | Wait-Process; Expand-Archive -Path "mdoc.zip" -DestinationPath "./mdoc/"
+        workingDirectory: $(Build.BinariesDirectory)
+        displayName: Download and Extract mdoc Zip
+      - task: NuGetToolInstaller@0
+        displayName: Install Nuget Client
+      - powershell: nuget install Microsoft.DocAsCode.ECMA2Yaml -ExcludeVersion
+        workingDirectory: $(Build.BinariesDirectory)
+        displayName: Install Microsoft.DocAsCode.ECMA2Yaml
       - powershell: |
           Invoke-WebRequest -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
           -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
         workingDirectory: $(Build.BinariesDirectory)
         displayName: Download and Extract DocFX
-      - task: CmdLine@2
-        displayName: Provision DocFX Directory
+      - powershell: ". '$(Build.SourcesDirectory)/eng/docgeneration/script/docgen.ps1';CreateDocGenFolders"
+        displayName: Create Directories for Doc Generation
+        workingDirectory: $(Build.BinariesDirectory)
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download published packages'
         inputs:
-          script: $(Build.BinariesDirectory)/docfx/docfx.exe init -q
-          workingDirectory: $(Build.SourcesDirectory)
-          failOnStderr: true
-      - powershell: ls
-        workingDirectory: $(Build.SourcesDirectory)/docfx_project/
-      - powershell: |
-          Copy-Item "$(Build.SourcesDirectory)/eng/docgeneration/docfx.json" -Destination "$(Build.SourcesDirectory)/docfx_project/" -Force
-        displayName: Copy over docfx.json
+          buildType: specific
+          project: '590cfd2a-581c-4dcb-a12e-6568ce786175'
+          definition: 37
+          buildVersionToDownload: latestFromBranch
+          tags: Scheduled
+          artifactName: packages
+          targetPath: '$(System.ArtifactsDirectory)/packages'
+      - powershell: ". '$(Build.SourcesDirectory)/eng/docgeneration/script/docgen.ps1';ExtractDlls -PkgDirPath $(System.ArtifactsDirectory)/packages -DllDirPath $(Build.BinariesDirectory)/dll-docs/my-api"
+        displayName: Extract dlls from packages and copy then to dlls directory
+      - powershell: nuget install NETStandard.Library -ExcludeVersion
```

Comment:
Once you have the wrapper project to do the restore you can also add NETStandard.Library and other nuget packages you need to the closure.

Reply:
Done